src/features/Login/components/Input.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Container,
+  FormContainer,
+  KakaoTitle,
+  InputForm,
+  LoginButton,
+} from './Input.styles'
+import { useNavigate, useLocation } from 'react-router-dom'
+import KaKaoTitleIcon from '@/assets/icons/kakao-title.svg?react'
+
+const Input: React.FC = () => {
+  const navigate = useNavigate()
+  const location = useLocation()
+  const hasPreviousPage = location.key !== 'default'
+
+  const handleLoginClick = () => {
+    if (hasPreviousPage) {
+      navigate(-1)
+    } else {
+      navigate('/')
+    }
+  }
+
+  return (
+    <Container>
+      <FormContainer>
+        <KakaoTitle>
+          <KaKaoTitleIcon />
+        </KakaoTitle>
+        <InputForm placeholder="이메일" type="emial"></InputForm>
+        <InputForm placeholder="비밀번호" type="password"></InputForm>
```

Comment:
이메일 오타가 있네요! 그리고 아래와 같이 수정할 수 있어보여요 😊

```suggestion
        <InputForm placeholder="이메일" type="email" />
        <InputForm placeholder="비밀번호" type="password" />
```

Reply:
> 이메일 오타가 있네요! 그리고 아래와 같이 수정할 수 있어보여요 😊

수정 완료